css-two-column-layout/styles.css

```diff
@@ -0,0 +1,34 @@
+.container {
+  font-family: sans-serif;
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  line-height: 26px;
+}
+
+.column-half {
```

Comment:
the column class should have a width applied to them. a half would indicate 50% width.